plezanje-net/web
# Area management tools

Tools for managing areas and merging crags / sectors / routes.

To test: 
- try to rename, create, delete areas via area editor: User icon > Administracija > Države in področja
- move a sector to another crag and see that activities and routes are also moved (sector list -> more button)
- to be extra tricky, try logging routes in two different sectors and then move one of the sectors - this should split the activity into two
- move route to another sector and merge it with an existing route, see that comments, votes and ascents are transferred accordingly (route list > more button)
